CONTRIBUTING.md

```diff
@@ -2,9 +2,11 @@
 
 1. Fork it!
 2. Create your feature branch: `git checkout -b my-new-feature`
-3. Commit your changes: `git commit -am 'Add some feature'`
-4. Push to the branch: `git push origin my-new-feature`
-5. Submit a pull request :D
+3. Test it: `cargo test`
+4. Lint it: `cargo +nightly clippy --all-targets -- -D warnings`
```

Comment:
How come we use nightly to lint but not nightly to test?

Reply:
nightly often has better lints, or lints that tell us about things that are about to become an issue. In GitHub Actions we test with beta, which is a bit of a tradeoff between stability and features. While for releases we release with stable. For contributors generally they can build with whatever they want, but I'd prefer the benefit of checking with the latest lints, and thats very easy to do.